paddle/fluid/framework/tensor.cc

```diff
@@ -62,8 +72,9 @@ void* Tensor::mutable_data(const platform::Place& place,
 
 void* Tensor::mutable_data(const platform::Place& place,
                            size_t requested_size) {
-  PADDLE_ENFORCE_NOT_NULL(
-      this->holder_, "Cannot invoke mutable data if current hold nothing.");
+  PADDLE_ENFORCE_NOT_NULL(this->holder_,
+                          platform::errors::PreconditionNotMet(
+                              "The `holder_` of Tensor is nullptr."));
```

Comment:
I think we can adjust the `holder_`, because users also doesn't know what is `holder_`, when holder is null, the tensor is not be initialized, maybe we can tell user `The tensor is not initialized.`?

Reply:
done, thx